pywbem/_logging.py

```diff
@@ -32,53 +32,103 @@
 
 * ``pywbem.ops`` logs user-issued calls to pywbem WBEMConnection methods that
   drive WBEM operations (see :ref:`WBEM operations`). Log
-  entries are created at INFO log level. This logs the parameters of each
+  entries are created at DEBUG log level. This logs the parameters of each
   request and the CIMObjects/exceptions in each method repoonse.
 
 * ``pywbem.http`` logs HTTP requests and responses between the pywbem client
-  and WBEM servers, at the info level. This logs the http request data and
+  and WBEM servers, at the DEBUG level. This logs the http request data and
   response data. This named logger is also defined to create log entries when
   logging is at the INFO log level.
 
-To output log records for one of the defined named loggers, either
-:meth:`~pywbem._logging.PywbemLoggers.create_loggers` or the
-:meth:`~pywbem._logging.PywbemLoggers.create_logger` should be used to define
-the characteristics of the named logger.
+The user can use the Python logger functionality to initiate either or both of
+these loggers or helper functions in the _logger module
```

Comment:
* We should not mention the (internal) "_logger" module at all.
* Not sure what "helper functions" are that are "initiated".
* I would rather state something like "can configure these loggers thrugh standard Python logger setup."